src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/WeekNum.txt

```diff
@@ -11680,7 +11677,7 @@ Error({Kind:ErrorKind.InvalidArgument})
 
 // Valid date passed as DateTimeValue parameter
 >> WeekNum(1/2/2022,StartOfWeek.Sunday)
```

Comment:
"Valid date passed as DateTimeValue" - this is incorrect. We are not passing a valid date here, we are passing a number (`1/2/2022` which is (1/2)/2022. I'm don't think it adds value to the test for this function itself.

Reply:
Thanks for catching this.